### DISCUSSION TITLE
Main Menu

### DISCUSSION THREAD

**Discussion by taydz** [taydz/Tafe230213#4] [General]
How is the layout of the main menu? is there anything you would change? or keep it the way it is ?

**Comment by kyronjavier** [comment]
I'm pretty happy with how the main menu looks, minimal so its easy to navigate. I would keep it as it is.

**Comment by taydz** [comment]
I'm thinking of keeping the colour theme from the original calculator, do you think we should use that?

**Reply by kyronjavier** [reply]
Definitely! Having one colour theme for all calculators would be nice.

**Comment by taydz** [comment]
I have finished with my calculator and I'm ready to make the merge to the development branch. So will start this process now. 
How are you going with your currency converter?

**Reply by kyronjavier** [reply]
Awesome,! I've also finished mine and can begin the merging. Let me know if you're done , so I can pull the latest development branch and merge mine.
